shellcheck validation of tasks/postgres_upgrade.sh fails

simple scripting tweak required:
```
$ shellcheck gitlab/tasks/postgres_upgrade.sh

In gitlab/tasks/postgres_upgrade.sh line 12:
  if [ $DB_SIZE -lt $FREE ]; then
       ^------^ SC2086: Double quote to prevent globbing and word splitting.
                    ^---^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
  if [ "$DB_SIZE" -lt "$FREE" ]; then

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```

--- a/tasks/postgres_upgrade.sh
+++ b/tasks/postgres_upgrade.sh
@@ -6,8 +6,8 @@ echo "version is ${CMD##* }"
 #9.2.18
 if [[ ${CMD##* } == "9.2.18" ]]; then
   echo 'Version is below required for Gitlab 10+, checking...'
-  DB_SIZE=$(du -shk /var/opt/gitlab/postgresql/data | awk '{print $1}')
-  FREE=$(df -hk /var/opt/gitlab/postgresql/data/ | tail -1 | awk '{print $4}')
+  DB_SIZE=$(du -sk /var/opt/gitlab/postgresql/data | awk '{print $1}')
+  FREE=$(df -k /var/opt/gitlab/postgresql/data/ | tail -1 | awk '{print $4}')
   echo "Database size is: $DB_SIZE kb and freespace is $FREE kb"
   if (( DB_SIZE < FREE )); then
     echo 'Enough freespace available to proceed.'

--- a/tasks/postgres_upgrade.sh
+++ b/tasks/postgres_upgrade.sh
@@ -4,12 +4,12 @@ echo 'Checking pgsql version'
 CMD=$(gitlab-psql --version)
 echo "version is ${CMD##* }"
 #9.2.18
-if [ "${CMD##* }" = "9.2.18" ]; then
+if [[ ${CMD##* } == "9.2.18" ]]; then
   echo 'Version is below required for Gitlab 10+, checking...'
   DB_SIZE=$(du -shk /var/opt/gitlab/postgresql/data | awk '{print $1}')
   FREE=$(df -hk /var/opt/gitlab/postgresql/data/ | tail -1 | awk '{print $4}')
   echo "Database size is: $DB_SIZE kb and freespace is $FREE kb"
-  if [ $DB_SIZE -lt $FREE ]; then
+  if (( DB_SIZE < FREE )); then
     echo 'Enough freespace available to proceed.'
     gitlab-ctl pg-upgrade
     echo 'Upgrade complete.  Please verify everything is correct and then run the post_upgrade task.'

